### DISCUSSION TITLE
zh_bronya下一个版本：NaN.NaN！

### DISCUSSION THREAD

**Discussion by YifanZhuZhu** [Moonlark-Dev/XDbot2#282] [General]
缺失：cave.error_to_download_images help.tips cave.cannot_comment cave.cannot_add mcver.mcver mcver.enabled mcver.disabled rule.permission_denied achi.unlocked chatgpt.proxy_set chatgpt.apikey_set chatgpt.cache_cleaned chatgpt.cache statistics.info qm_avg.info bag.item_info cave.not_allow_null_cave pouch.got pouch.not_enough_space bank.usage pouch.put currency.no_money currency.ok bank.full bank.item bank.view pouch.help currency.unknown_argv github.open_url github.login_successful towel_zip.used email.email email.items email.remind email.id email.all_read dice.20 dice.18..19 dice.15..17 dice.10..14 dice.2..9 dice.1-1 dice.1-2 dice.else update-notice.enabled update-notice.disabled groupwelcome.group.enabled groupwelcome.group.disabled groupwelcome.global.enabled groupwelcome.global.disabled groupwelcome.application.enabled groupwelcome.application.disabled groupwelcome.message.wm.edited groupwelcome.message.aam.edited groupwelcome.message.anm.edited groupwelcome.config.reset essencemsg.keyword.added essencemsg.keyword.removed asc.actived asc.inactived asc.signed_1 asc.signed_2 asc.signed_3

**Comment by This-is-XiaoDeng** [comment]
可还行（

**Comment by This-is-XiaoDeng** [comment]
你准备什么时候更新呢（x

**Comment by This-is-XiaoDeng** [comment]
随时欢迎（

**Comment by This-is-XiaoDeng** [comment]
zh_bronya 是不是需要同步一下了（
